### ISSUE TITLE
WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'

### ISSUE THREAD

**Issue by JenuJari** [facebook/fbt#179]
## 💬 Having warning in the Gradle build flow. 

I am having the following log printed when I do Gradle build for my react native android project. 

```
Configure project :react-native-fbt
WARNING: API 'variant.getJavaCompile()' is obsolete and has been replaced with 'variant.getJavaCompileProvider()'.
It will be removed in version 5.0 of the Android Gradle plugin.
For more information, see https://d.android.com/r/tools/task-configuration-avoidance.
To determine what is calling variant.getJavaCompile(), use -Pandroid.debug.obsoleteApi=true on the command line to display more information.
```

Is it ok to ignore this or have some extra settings to do in my setup?

**Comment by jrwats** [comment]
cc @dalmendray

**Comment by JenuJari** [comment]
Do we need some more info?

**Comment by jrwats** [comment]
@JenuJari , this should be fixed as of 3388738c47660f1188f6c24e3344692784f9fabb and in `react-native-fbt` `v0.0.2`
https://www.npmjs.com/package/react-native-fbt
